core/templates/dev/head/pages/exploration-editor-page/settings-tab/settings-tab.directive.ts

```diff
@@ -84,9 +87,9 @@ angular.module('oppia').directive('settingsTab', [
       controllerAs: '$ctrl',
       controller: [
         '$http', '$rootScope', '$scope', '$uibModal', '$window',
-        'AlertsService',
-        'ChangeListService', 'EditableExplorationBackendApiService',
-        'EditabilityService', 'ExplorationAutomaticTextToSpeechService',
+        'AlertsService', 'ChangeListService',
+        'EditableExplorationBackendApiService', 'EditabilityService',
```

Comment:
Thanks for fixing this can you please also add then in sorted order?

Reply:
Done

Reply:
is this resolved?

Reply:
I somehow missed it yesterday.
Done.